addons/product/views/product_views.xml

```diff
@@ -341,6 +341,12 @@
                 <xpath expr="//div[hasclass('oe_title')]" position="inside">
                     <field name="attribute_value_ids" widget="many2many_tags" groups="product.group_product_variant"/>
                 </xpath>
+                <xpath expr="//label[@for='standard_price']" position="attributes">
+                    <attribute name="attrs">{}</attribute>
+                </xpath>
+                <xpath expr="//div[@name='standard_price_uom']" position="attributes">
+                    <attribute name="attrs">{}</attribute>
+                </xpath>
```

Comment:
feels complicated.  Does not it conflict with this thing in stock_account: 

<field name="standard_price" position="replace">
                        <div class="o_row">
                            <field name="standard_price"
                                attrs="{'readonly':[('valuation','=','real_time')]}"
                                widget='monetary' options="{'currency_field': 'currency_id', 'field_digits': True}" />
                            <button string="Update Cost" type="action"
                            name="%(action_view_change_standard_price)d"
                            attrs="{'invisible':['|', '|', ('valuation','!=', 'real_time'), ('cost_method', 'not in', ['average', 'standard']), ('product_variant_count', '>', 1)]}"
                            class="oe_link oe_read_only" colspan="2"/>
                        </div>
                    </field>

Reply:
with this you mean: https://github.com/odoo/odoo/blob/11.0/addons/stock_account/views/product_views.xml#L40 ?
I don't see how it feels complicated. Yes, actually, so I will have to find another solution.